web/src/scheduled_messages_popover.js

```diff
@@ -17,6 +17,7 @@ import {parse_html} from "./ui_util";
 import {user_settings} from "./user_settings";
 
 export const SCHEDULING_MODAL_UPDATE_INTERVAL_IN_MILLISECONDS = 60 * 1000;
+const ENTER_SENDS_SELECTION_DELAY = 600;
```

Comment:
@terpimost @alya FYI this 600ms delay when you click a different option in the "Enter sends" popover is sorta a unique value in the app. I don't have a clear idea in my mind what this would want to be consistent with or how we want feedback for selecting that option in this popover to work, exactly (In a lot of other places, we just close the popover on click), but I think we can settle this as a follow-up.